src/app/register/register.component.html

```diff
@@ -1,38 +1,78 @@
-        <p *ngIf="regForm.controls['email'].hasError('pattern') && (regForm.controls['email'].touched) && !isFocused" class="alert alert-danger" role="alert">{{authMessages?.errors.emailInvalid}}</p>
-        <p *ngIf="regForm.controls['password'].hasError('required') && (regForm.controls['password'].touched && !isFocused)" class="alert alert-danger" role="alert">{{authMessages?.errors.passwordReq}}</p>
-        <p *ngIf="regForm.controls['confirmpassword'].value!=regForm.controls['password'].value && (regForm.controls['confirmpassword'].touched && !isFocused)" class="alert alert-danger" role="alert">{{authMessages?.errors.passwordMatch}}</p>
-      </div>
+<mat-toolbar id="header" class="row">
+    <div class="col-md-8 col-md-offset-2 text-center">
+      <span class="header-white-1 text-center">{{header?.headings.titleLeft}}</span><span class="header-2 text-center">{{header?.headings.titleRight}}</span>
+    </div>
+    <div class="col-md-2">
+      <img src="assets/img/leftBanner.png" id="banner">
+    </div>
+</mat-toolbar>
+
+
+<div class="col-xs-8 col-xs-offset-2 text-center">
+  <h3 class="form-signin-heading">{{language?.messages.heading}}</h3>
+
+  <form [formGroup]="regForm" (ngSubmit)="onSubmit(regForm.value)" id="registration">
+    <div class="form-container">
+      <mat-form-field appearance="standard">
+        <mat-label>{{language?.labels.firstName}}</mat-label>
+        <mat-icon matSuffix>person</mat-icon>
+        <input matInput name="fname" placeholder={{language?.placeHolders.firstName}}
+               [formControl]="regForm.controls['fname']">
+        <mat-error *ngIf="regForm.controls['fname'].hasError('required')">{{authMessages?.errors.firstNameReq}}
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field appearance="standard">
+        <mat-label>{{language?.labels.lastName}}</mat-label>
+        <mat-icon matSuffix>person</mat-icon>
+        <input matInput name="lname" placeholder={{language?.placeHolders.lastName}}
+               [formControl]="regForm.controls['lname']">
+      </mat-form-field>
+      <mat-form-field appearance="standard">
+        <mat-label>{{language?.labels.email}}</mat-label>
+        <mat-icon matSuffix>mail_outline</mat-icon>
+        <input matInput name="email" placeholder={{language?.placeHolders.email}}
+               [formControl]="regForm.controls['email']">
+        <mat-error *ngIf="regForm.controls['email'].hasError('required')">{{authMessages?.errors.emailReq}}</mat-error>
+        <mat-error *ngIf="regForm.controls['email'].hasError('pattern')">{{authMessages?.errors.emailInvalid}}
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field appearance="standard">
+        <mat-label>{{language?.labels.password}}</mat-label>
+        <mat-icon matSuffix (click)="hide = !hide">{{hide ? 'visibility' : 'visibility_off'}}</mat-icon>
```

Comment:
Can this mat-icon change to a pointer type cursor on hovering over it?
Same for the confirm password field as well